recipes/python-neo-lzf/recipe.yaml

```
@@ -0,0 +1,47 @@
+context:
+  name: python-neo-lzf
+  version: "0.3.2"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/FledgeXu/${{ name }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 3054f2938d8d55f0196068d3bceaf76b38dc3084ba1dc836c4b571a11ba00125
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
```

```suggestion
    - ${{ compiler('c') }}
    - ${{ stdlib('c') }}
    - if: (build_platform != target_platform)
       then:
         - cross-python_${{ target_platform }}
         - python
```
This is needed for proper cross-compilation support for native Python packages.